css/src/style.scss

```diff
@@ -327,7 +326,7 @@ $white: #fff;
 		margin: 12px 0;
```

Comment:
You ca use the new add button standard! :) 
https://github.com/nextcloud/server/blob/master/core/css/apps.scss#L99
```html 
<!-- app-navigation -->
<div class="app-navigation-new">
<button type="button" id="new-user-button" class="icon-add">New user</button>
</div>
<!-- ul -->
```

Reply:
You mean to use the new button to create new tasks?

Reply:
Yes. You can look the 14 documentation. On the design section. :)